### ISSUE TITLE
Update Color Contrast - Footer

### ISSUE THREAD

**Issue by c-deems** [alaoweb/2021.alaoweb.org#99]
Per the Pa11y Dashboard, elements in the footer have been noted as having insufficient contrast at their current level. A contrast ratio of at least 4.5:1 is needed, but the text currently has a contrast ratio of 4.49:1. The dashboard recommends changing the either the footer or website background to #fdfdfd, though changing the text color may work as well.

The following lines were specifically identified as needing to be updated to meet WCAG AA compliance:
`#footer > div:nth-child(1) > div:nth-child(1) > ul > li > a
#footer > div:nth-child(1) > div:nth-child(2) > ul > li > a
#footer > div:nth-child(1) > div:nth-child(3) > ul > li > a
#footer > div:nth-child(3) > div > p > a:nth-child(1)
#footer > div:nth-child(3) > div > p > a:nth-child(2)
#footer > div:nth-child(3) > div > p > a:nth-child(3)`
